docs/docs/rest-api/public/api/v2/types/resources.raml

```diff
@@ -0,0 +1,30 @@
+#%RAML 1.0 DataType
+type: object
+description: Resource Allocations
+properties:
+  cpus:
+    displayName: cpu shares
+    description: The number of CPU shares this pod needs per instance. This number does not have to be integer, but can be a fraction.
+    type: number
+    minimum: 0
```

Comment:
should probably be 0.001 (the lowest value that has meaning given fixed precision scalars in mesos)